priv/docker-compose.geth-ganache.yml

```diff
@@ -11,7 +11,7 @@ services:
       - ETHEREUM_WS_RPC_URL=ws://geth:8545
   plasma-contracts:
     environment:
-      - MIN_EXIT_PERIOD=1
+      - MIN_EXIT_PERIOD=22
```

Comment:
How does this relate to `@min_exit_period` in `Itest.InFlightExitClient` ? Is the goal to have `@min_exit_period` less than `MIN_EXIT_PERIOD` so the in-flight exit can finish in time? If so, do we need to set it this high or was `22` just a random number higher than `1` to get tests passing?

Reply:
I synced with @piotr on this and he recommended for IFE that the exit period be set at 22. This allows it enough time for it to also participate in the "first phase of exit", e.g piggybacking I believe.

Reply:
Well, 22 worked for `elixir-omg`-based tests. I think the number was roughly pinned down by trial-error, for a specific setup. If this new setup allows for tighter values you can test it out. The less, the better, as long as the test isn't flaky.